pandas/tests/indexes/multi/test_contains.py

```diff
@@ -57,6 +58,23 @@ def test_isin():
     assert len(result) == 0
     assert result.dtype == np.bool_
 
+    msg_tmpl = 'Value length must be equal to count of levels. len(%s) != %d'
+    levcnt = len(idx.levels)
+
+    # some values too short
+    vals_short = [('foo',), ('bar', 3), ('quux',)]
+    msg = re.escape(msg_tmpl % (vals_short[0], levcnt))
+
+    with pytest.raises(ValueError, match=msg):
+        idx.isin(vals_short)
+
+    # some values too long
+    vals_long = [('foo', 2), ('bar', 3, 2), ('quux', 4)]
+    msg = re.escape(msg_tmpl % (vals_long[1], levcnt))
+
+    with pytest.raises(ValueError, match=msg):
+        idx.isin(vals_long)
+
```

Comment:
can you parametrise these two cases

Reply:
Not quite sure how to do this. Following this [pandas pytest guide](https://github.com/pandas-dev/pandas/blob/master/doc/source/development/contributing.rst#using-pytest), I broke the raise tests into a separate `test_isin_values_raises` function, then added two sets of parameters (one for too short and one for too long args).

Let me know if that's right.

Reply:
> Let me know if that's right.

generally looks good. we don't need a separate module for this though, where you had it before is fine.

Reply:
sorry looking at the wrong diff. was expecting to see the changes. here is fine